### PR TITLE
feat: Looker 21.4 bindings

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

**Comment by joeldodge79** [comment]
an important thing to decide about this PR is the title. If merged as it currently exists (`feat: Looker 21.4 bindings`) it will generate the following changelog entries for typescript and python (we'll have to manually update the non release-please managed languages till I can get them added to release-please):

```md
### Features

* Looker 21.4 bindings ([abcdefg](https://www.github.com/looker-open-source/sdk-codegen/commit/abcdefg))
```

File: packages/sdk-codegen-scripts/src/sdkConfig.ts [hunk 1]
```diff
@@ -70,5 +70,6 @@ export const SDKConfig = (
   if (!config.base_url) {
     throw Error('Fatal error: base_url is not configured. Exiting.')
   }
+  console.log(`BASE URL: ${config.base_url}`)
```
Comment (joeldodge79) [inline packages/sdk-codegen-scripts/src/sdkConfig.ts:73]: oops
```suggestion
```

File: packages/sdk-node/src/nodeTransport.ts [hunk 2]
```diff
@@ -172,6 +172,7 @@ export class NodeTransport extends BaseTransport {
     options?: Partial<ITransportSettings>
   ): Promise<SDKResponse<TSuccess, TError>> {
     try {
+      console.log(`PATH: ${path}`)
```
Comment (joeldodge79) [inline packages/sdk-node/src/nodeTransport.ts:175]: oops
```suggestion
```

**Review by jkaster — APPROVED** [review]
LGTM. Thanks for catching those errant console.log calls

**Pull request merged**